### ISSUE TITLE
`Cannot redeclare block-scoped variable 'console'.` error when `@types/node` is installed

### ISSUE THREAD

**Issue by yuanqing** [yuanqing/create-figma-plugin#126]
```
$ npx -- create-figma-plugin --yes --template plugin/hello-world
$ cd hello-world
$ npm i --save-dev @types/node
$ npm run build
```

**Comment by ben-katz** [comment]
Any temporary fix for this?

**Comment by yuanqing** [comment]
Fixed in `2.1.4`

**Comment by ben-katz** [comment]
@yuanqing how should I update to 2.1.4?
